client/signup/steps/p2-complete-profile/index.jsx

```diff
@@ -0,0 +1,163 @@
+			setIsSubmitting( false );
+
+			if ( ! userSettingsRequestFailed ) {
+				recordTracksEvent( 'calypso_signup_p2_complete_profile_step_submit' );
+
+				const stepData = {
+					stepName: stepName,
+					formFullName,
+				};
+
+				submitSignupStep( stepData );
+
+				goToNextStep();
+			}
+		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [ isSubmitting, updatingUserSettings, userSettingsRequestFailed ] );
+
+	const renderUploadAvatarBtn = () => {
+		return (
+			<button className="p2-complete-profile__upload-avatar-btn">
+				{ translate( 'Upload a new avatar' ) }
+			</button>
+		);
+	};
+
+	const handleFormSubmit = ( event ) => {
+		event.preventDefault();
+
+		setIsSubmitting( true );
+
+		setFormErrors( {} );
+
+		if ( formFullName.length < 2 ) {
+			setFormErrors( {
+				fullName: translate( 'Please enter your full name (3 characters or more).' ),
+			} );
+
+			setIsSubmitting( false );
+
+			return;
+		}
+
+		// API call to update user profile.
+		dispatch( saveUserSettings( { display_name: formFullName } ) );
+	};
+
+	const handleSkipBtnClick = () => {
+		submitSignupStep( {
+			stepName: stepName,
+		} );
+
+		recordTracksEvent( 'calypso_signup_p2_complete_profile_skip_button_click' );
+
+		goToNextStep();
+	};
+
+	return (
+		<P2StepWrapper
+			flowName={ flowName }
+			stepName={ stepName }
+			positionInFlow={ positionInFlow }
+			headerText={ translate( 'Complete your profile' ) }
+			subHeaderText={ translate(
+				'Using a recognizable photo and name will help your team to identify you more easily.'
+			) }
+		>
+			<div className="p2-complete-profile">
+				<div className="p2-complete-profile__avatar-wrapper">
+					<EditGravatar additionalUploadHtml={ renderUploadAvatarBtn() } />
+				</div>
+
+				<div className="p2-complete-profile__form-wrapper">
+					<form className="p2-complete-profile__form" onSubmit={ handleFormSubmit } noValidate>
+						<label htmlFor="full-name-input" className="p2-complete-profile__form-label form-label">
+							{ translate( 'Your Full Name' ) }
+						</label>
+						<input
+							type="text"
+							id="full-name-input"
+							name="full-name"
+							className="p2-complete-profile__full-name form-text-input"
+							disabled={ isSubmitting }
+							value={ formFullName }
+							onChange={ ( event ) => setFormFullName( event.target.value ) }
+						/>
+						{ formErrors?.fullName && <FormInputValidation text={ formErrors.fullName } isError /> }
+						<div className="p2-complete-profile__form-footer">
+							<Button
+								type="submit"
+								variant="primary"
+								className="p2-complete-profile__form-submit-btn"
+								disabled={ isSubmitting }
+							>
+								{ translate( 'Continue' ) }
+							</Button>
+						</div>
+					</form>
+				</div>
+				<div className="p2-complete-profile__skip-wrapper">
+					{ createInterpolateElement(
+						__( 'No time? No problem! You can <Button>do this later</Button>.' ),
```

Comment:
I find it very odd that we use both `i18n-calypso` and `@wordpress/i18n` for translations. Can we use just one of them in a single component?